lighthouse/Cargo.toml

```diff
@@ -16,6 +16,8 @@ modern = ["bls/supranational-force-adx"]
 milagro = ["bls/milagro"]
 # Support minimal spec (used for testing only).
 spec-minimal = []
+# Support gnosis spec and gnosis beacon chain.
```

Comment:
```suggestion
# Support Gnosis spec and Gnosis Beacon Chain.
```

Dummy edit to trigger CI again

Reply:
Ah, I can't commit this. Will need to wait for you Kirill :pray:

Reply:
I've committed this, but not sure if it helped. Do you see any changes regarding CLA?